### ISSUE TITLE
inpout driver is a security risk.

### ISSUE THREAD

**Issue by alberts8** [irusanov/ZenTimings#38]
As long as the inpout driver used by ZenTimings is installed it allows universal read-write access to the whole systems memory. This is especially problematic because it enables software running without admin privileges to do that.

**Comment by alberts8** [comment]
Only having the driver installed while Zen Timings is running would be a improvement. Because currently after you use Zen Timings the driver stays installed.

The developers of that driver could have easily used the WdmlibIoCreateDeviceSecure api to make administrative privileges required to use the driver but they didn't.

**Comment by irusanov** [comment]
Yes, I have the code for uninstallation and cleanup on close, but it is still problematic if some other app is using the driver at the same time. Will think about it.

**Comment by irusanov** [comment]
@alberts8 I'm going to try remove the original DLL and implement an own installer of the kernel driver and service.
